patient_issue_finder.sh

```diff
@@ -0,0 +1,131 @@
+#!/bin/bash
+#This  script runs  in all pattners databases in order  to find nonconformity patient 
+# and  displays in in *.csv  file
+
+
+#Collecting MySQL credential
+echo "Collecting MySQL credentials........................................................................"
+echo "MySQL User: "
+read mysql_user
+echo "MySQL Password: "
+read -s mysql_passwd
+echo "MySQL IP: "
+read host
+
+#creating the folder "/epts_issue_data_finder"  in home user  directory if  it  does not exist
+echo "Creating the  'epts_issue_data_finder' folder  in home user directory if  it  does not exist........"
+if [ ! -d "~/epts_issue_data_finder" ];
+	then 
+		mkdir ~/epts_issue_data_finder
+fi
+
+#wipe all data in the folder before running
+rm  -rf ~/epts_issue_data_finder/*
+
+
+#choosing the running  approach
+echo "Do you  want to  perform this  script in all databases? [y/n] (lower case please)"
+read all_db_anserwer
+
+#declaring an empty array to store the databases names 
+declare -a  db_list=()
+
+
+if [ $all_db_anserwer == "y" ]
+	then
+    		mysql -u $mysql_user -p$mysql_passwd -h $host -e "show databases;" >  ~/epts_issue_data_finder/temp_file.txt
+
+                readarray db_list   <  ~/epts_issue_data_finder/temp_file.txt
```

Comment:
is readarray found in all os where we could run this! is it in OSX forexample?

Reply:
need to  check an alternative way to  do that, but this  script was intended to run  in  Linux  servers...But  ok,  in order to allow you to check I will see another  way